crates/provider/src/docker/node.rs

```diff
@@ -0,0 +1,527 @@
+                            "/scripts".to_string(),
+                        ),
+                        (
+                            format!("{}-helper-binaries", self.namespace_name()),
+                            "/helpers".to_string(),
+                        ),
+                        (
+                            self.config_dir.to_string_lossy().into_owned(),
+                            "/cfg".to_string(),
+                        ),
+                        (
+                            self.data_dir.to_string_lossy().into_owned(),
+                            "/data".to_string(),
+                        ),
+                        (
+                            self.relay_data_dir.to_string_lossy().into_owned(),
+                            "/relay-data".to_string(),
+                        ),
+                    ]))
+                    .entrypoint("/scripts/zombie-wrapper.sh"),
+            )
+            .await
+            .map_err(|err| ProviderError::NodeSpawningFailed(self.name.clone(), err.into()))?;
+
+        Ok(())
+    }
+
+    async fn initialize_db_snapshot(
+        &self,
+        db_snapshot: &AssetLocation,
+    ) -> Result<(), ProviderError> {
+        todo!()
+        // trace!("snap: {db_snapshot}");
+        // let url_of_snap = match db_snapshot {
+        //     AssetLocation::Url(location) => location.clone(),
+        //     AssetLocation::FilePath(filepath) => self.upload_to_fileserver(filepath).await?,
+        // };
+
+        // // we need to get the snapshot from a public access
+        // // and extract to /data
+        // let opts = RunCommandOptions::new("mkdir").args([
+        //     "-p",
+        //     "/data/",
+        //     "&&",
+        //     "mkdir",
+        //     "-p",
+        //     "/relay-data/",
+        //     "&&",
+        //     // Use our version of curl
+        //     "/cfg/curl",
+        //     url_of_snap.as_ref(),
+        //     "--output",
+        //     "/data/db.tgz",
+        //     "&&",
+        //     "cd",
+        //     "/",
+        //     "&&",
+        //     "tar",
+        //     "--skip-old-files",
+        //     "-xzvf",
+        //     "/data/db.tgz",
+        // ]);
+
+        // trace!("cmd opts: {:#?}", opts);
+        // let _ = self.run_command(opts).await?;
+
+        // Ok(())
+    }
+
+    async fn initialize_startup_files(
+        &self,
+        startup_files: &[TransferedFile],
+    ) -> Result<(), ProviderError> {
+        try_join_all(
+            startup_files
+                .iter()
+                .map(|file| self.send_file(&file.local_path, &file.remote_path, &file.mode)),
+        )
+        .await?;
+
+        Ok(())
+    }
+
+    pub(super) async fn start(&self) -> Result<(), ProviderError> {
+        self.docker_client
+            .container_exec(
+                &self.container_name,
+                vec!["sh", "-c", "echo", ">", "/tmp/zombiepipe"],
```

Comment:
```suggestion
                vec!["sh", "-c", "echo start > /tmp/zombiepipe"],
```

Reply:
In other methods we are just using `["echo", "<method>", ">",  "/tmp/zombiepipe"] we should change here also.